src/сomponents/withLoadingDelay/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react'
+import { Spinner, Box } from '@chakra-ui/react'
+
+function withLoadingDelay(Component) {
+    function HOC(props) {
```

Comment:
Use arrow notation for all your functions. You should try to use one style everywhere.